cpp/src/parquet/encoding_test.cc

```diff
@@ -665,6 +666,33 @@ class EncodingAdHocTyped : public ::testing::Test {
     ::arrow::AssertArraysEqual(*values, *result, /*verbose=*/true);
   }
 
+  void PlainTwice(int seed) {
```

Comment:
Is it possible to extend `void Plain(int seed)` to be something like `void Plain(int seed, int round = 1)` to avoid duplicate code?

Reply:
That would be better, I'll do like this tomorrow.